source/docs/03-metadata.xml

```diff
@@ -1160,7 +1160,7 @@
                            <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample063.txt" parse="text"/></egXML>
                         </figure>
                      </p>
-                     <p>Music for a single player should, however, never use the <att>solo</att> attribute.</p>
```

Comment:
Didn't we want to keep this as a warning?

Reply:
I can't find it for now, but I'm sure that we discussed this. For me, this is 'just' a best practice suggestion. I we keep it I would suggest rephrasing it, e.g., `Music for a single player is not needed to be indicated as solo using the <att>solo</att> attribute.`

Reply:
Yes, i think we discussed it with the group in the Wannsee room. Maybe `Music for a single player does not have to be marked as solo with the <att>solo</att> attribute.`

Reply:
@musicEnfanthen Done

Reply:
Thank you!